### PR TITLE
[ODS-5415] Change Queries Enhancements for v5.3 - update open API metadata

### PR DESCRIPTION
**Pull request by AxelMarquez**

### CODE REVIEW

File: Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs [hunk 1]
```diff
@@ -406,7 +422,7 @@ private Operation CreateDeletesOperation(OpenApiMetadataPathsResource openApiMet
                     new Response
                     {
                         description =
-                            "Gone. An attempt to connect to the database for the snapshot specified by the Snapshot-Identifier header was unsuccessful (indicating the snapshot may have been removed)."
+                                                        "Gone. An attempt to connect to the database for the snapshot specified by the Snapshot-Identifier header was unsuccessful (indicating the snapshot may have been removed)."
```
Comment (gmcelhanon) [inline Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs:425]: Whitespace change is a bit odd here. Can we reformat (or revert) this? Was this formatted this way by Resharper?

File: Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs [hunk 2]
```diff
@@ -440,6 +456,63 @@ private Operation CreateDeletesOperation(OpenApiMetadataPathsResource openApiMet
...
+            };
+
+            if (_apiSettings.IsFeatureEnabled(ApiFeature.ChangeQueries.GetConfigKeyName()))
+            {
+                responses.Add(
+                    "410",
+                    new Response
+                    {
+                        description =
+                                                        "Gone. An attempt to connect to the database for the snapshot specified by the Snapshot-Identifier header was unsuccessful (indicating the snapshot may have been removed)."
```
Comment (gmcelhanon) [inline Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs:481]: Can you fix the formatting here too?

File: Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs [hunk 3]
```diff
@@ -440,6 +456,63 @@ private Operation CreateDeletesOperation(OpenApiMetadataPathsResource openApiMet
             };
         }
```
Comment (gmcelhanon) [inline Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs:457]: I know the verbiage for the deletes child resource was pre-existing, but can you rephrase the description to say "This operation is used to retrieve identifying information about resources that have been deleted."

File: Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs [hunk 4]
```diff
@@ -440,6 +456,63 @@ private Operation CreateDeletesOperation(OpenApiMetadataPathsResource openApiMet
...
+
+            return new Operation
+            {
+                tags = new List<string>
+                {
+                    OpenApiMetadataDocumentHelper.GetResourcePluralName(openApiMetadataResource.Resource)
+                        .ToCamelCase()
+                },
+                summary = "Retrieves resources key changes based on change version.",
+                description = "The operation is used to retrieve resources key changes.",
```
Comment (gmcelhanon) [inline Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs:502]: Change this description to: "This operation is used to retrieve identifying information about resources whose key values have been changed."

**Review by gmcelhanon — CHANGES_REQUESTED** [review]
A few minor changes. Overall, looks good.

File: Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs (hunk 1 above)
Comment (AxelMarquez) [reply Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs]: I thought that it was reformatted by Resharper, but I triggered the formatting again and I couldn't reproduce it, probably I inadvertently changed it.
Reverted it.

**Review by gmcelhanon — APPROVED** [review]
